ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/user_management_service.proto

```diff
@@ -148,6 +153,9 @@ message Right {
         string party = 1;
     }
 
+    // The right to administrate the identity provider.
+    message IdentityProviderAdmin {}
```

Comment:
Does this right exclusively grant access to the IdentityProviderConfigService; or does it have any additional implications? 

Is `IdentityProviderAdmin` 1) disjoint from or 2) a superset of `ParticipantAdmin` right?

Reply:
It has additional implications. IdentityProviderAdmin can manage users within his identity provider configuration. I.e. users will be assigned to a specific configuration and will be isolated from other users. So implication will be also spread to user management service (see an updated proto doc)